### PR TITLE
feat(table): table horizontal scrolling + Tag style fix

### PR DESCRIPTION
**Pull request by masoudmanson**
Horizontal Scrolling for the Table Component + Some style fixes

### CODE REVIEW

File: src/core/CellComponent/index.tsx [hunk 1]
```diff
@@ -14,7 +14,9 @@ const CellComponent = forwardRef(
     const { children } = props;
 
     return (
-      <StyledCellComponentData {...props}>{children}</StyledCellComponentData>
+      <StyledCellComponentData className="cell-component" {...props}>
```
Comment (bethbertozzi) [inline src/core/CellComponent/index.tsx:17]: thanks for catching this

Comment (masoudmanson) [reply src/core/CellComponent/index.tsx:17]: When the TableRow is disabled, I needed this className to properly style the cell components!

**Review by clarsen-czi — CHANGES_REQUESTED** [review]
Looking good! Just a small change to make on the margin on the TagFilter component and then we should be good.

**Review by clarsen-czi — APPROVED** [review]
Looks good!

**Pull request merged**